src/recommendations/src/recommendations-service/app.py

```diff
@@ -47,7 +49,9 @@
 sts = boto3.client('sts')
 cw_events = boto3.client('events')
 
-# SSM parameter name for the Personalize filter for purchased items
+# SSM parameter names
+product_recs_param_name = 'retaildemostore-product-recommendation-campaign-arn'
+related_products_param_name = 'retaildemostore-related-products-campaign-arn'
```

Comment:
We should start moving to the AWS-recommended path naming '/' convention for SSM parameters if possible